mtags/src/main/scala/scala/meta/internal/pc/Keyword.scala

```diff
@@ -81,7 +83,15 @@ object Keyword {
     Keyword("using", isParam = true, isScala3 = true),
     Keyword("var", isBlock = true, isTemplate = true, isDefinition = true),
     Keyword("given", isBlock = true, isTemplate = true, isDefinition = true, isScala3 = true),
-    Keyword("extension", isBlock = true, isTemplate = true, isDefinition = true, isPackage = true, isScala3 = true),
+    Keyword(
+      "extension",
+      isBlock = true,
+      isTemplate = true,
+      isDefinition = true,
+      isPackage = true,
+      isScala3 = true,
+      reversedTokensPredicate = Some(!extendsPred(_))
```

Comment:
It's mostly about adding this, so we don't show `extension` where we would expect `extend`